examples/vortex.py

```diff
@@ -56,6 +51,12 @@
         (1, 0, 2),
         )
 
+#######################################################################
+# Finally, we create a viewer, add the vortex frames, an overlay for the flow
+# magnitude, and the vectors field.
```

Comment:
```suggestion
# Finally, we create a viewer, and add the vortex frames, the flow
# magnitude, and the vector field.
```

"Overlay" (noun) is now a loaded word in napari that means something specific, so I think it's best to not use the term here, since it's just a normal layer.